vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -22510,13 +22510,17 @@ unit:MegaYR
     a qudt:DerivedUnit ;
     qudt:plainTextDescription "1,000,000-fold of the derived unit year." ;
     qudt:conversionMultiplier 1000000.0 ;
```

Comment:
Is this the correct multiplier? It should be relative to the SI unit for this dimensionality - i.e. seconds.

Reply:
Ooops, assumed it was relative to the YR but obviously how could it be? Have changed it to 1M x YR = 31557600000000.0